common/expirecontainer.h

```diff
@@ -210,6 +210,8 @@ class ExpireList : public ExpireContainer<T> {
     }
 };
 
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
what is this for?

Reply:
Split expire container and object cache

Reply:
Why split? `ObjectCache` is a derived class of `ExpireContainerBase`.

Maybe put all base classes to front and let `ExpireContainer` `ObjectCache` stay together is a better idea.